storage/src/objects/insert_commit.rs

```diff
@@ -283,6 +283,15 @@ impl<T: TransactionScheme, P: LoadableMerkleParameters, S: Storage> Ledger<T, P,
             value: block_header_hash.0.to_vec(),
         });
 
+        // If it's the genesis block, store its initial applicable digest.
+        if is_genesis {
+            database_transaction.push(Op::Insert {
+                col: COL_DIGEST,
+                key: to_bytes![self.current_digest()?]?.to_vec(),
+                value: 0u32.to_le_bytes().to_vec(),
```

Comment:
note: this means we'll now have 2 digests that point to block `0` if there are any transactions in the genesis block.